src/abstractdatatable/reshape.jl

```diff
@@ -193,30 +188,17 @@ function unstack(dt::AbstractDataTable, rowkey::Int, colkey::Int, value::Int)
     # `rowkey` integer indicating which column to place along rows
     # `colkey` integer indicating which column to place along column headers
     # `value` integer indicating which column has values
-    refkeycol = NullableCategoricalArray(dt[rowkey])
```

Comment:
Please keep the old code here. Just like for `join`, we cannot be sure that there won't be any missing values, so we need to create nullable columns. Cf. [my previous comment](https://github.com/JuliaData/DataTables.jl/pull/30#discussion_r105493256).

Same for `unstack`.